app/helpers/filters_helper.rb

```diff
@@ -42,6 +43,11 @@ def organisations_filter_options(user)
     [OpenStruct.new(id: "", name: "Select an option")] + organisation_options.map { |org| OpenStruct.new(id: org.id, name: org.name) }
   end
 
+  def assigned_to_filter_options(user)
+    user_options = user.organisation.users
+    [OpenStruct.new(id: "", name: "Select an option")] + user_options.map { |user_option| OpenStruct.new(id: user_option.id, name: user_option.name) }
+  end
+
```

Comment:
Do we want this to show all the users for support user?

Reply:
Hm good point - I imagine so but I'll check and update the ACs to include that if so :)

Reply:
have made this change, just waiting to hear back re whether we should also include non-support users' managing agents'/stock owners' users

Reply:
we should, and now we do ^